spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -2798,11 +2795,10 @@ def keyPressEvent(self, event):
                 self.completion_text += text
 
     def handle_close_parentheses(self, text):

```

Comment:
Let's change this method to `handle_parentheses` instead. I think it's a more appropriate name given the changes you made.